layouts/shortcodes/include-sphinx-html.html

```diff
@@ -0,0 +1,6 @@
+{{ $file := .Get 0 }}
+{{ $content := readFile $file }}
+    <link rel="stylesheet" type="text/css" href="/automatically_generated_oak_cli_docs/_static/pygments.css" />
```

Comment:
Personal taste, I feel the in light mode is rendering too light and with low contrast. Maybe a slightly darker grey will do the trick?
 
<img width="706" alt="image" src="https://github.com/user-attachments/assets/d179c27d-085d-484b-9ebe-1254f10eeadf">

Reply:
I second this! The output shows nicely in dark mode but in light mode the text colors need to darken

Reply:
I will investigate this now because on my end this looks way different:

![image](https://github.com/user-attachments/assets/f4d538fa-639f-40c0-9f62-c01347c8c78e)


![image](https://github.com/user-attachments/assets/5d1b1552-7046-48b1-9320-ed2c4782d50d)

Reply:
I just had a tiny debugging session with Giovanni - turns out running "npm install" fixed the issues